### PR TITLE
WIP: Modified job definition yaml to template style

### PR DESCRIPTION
**Pull request by ScrewTSW**

### CODE REVIEW

**Comment by ScrewTSW** [comment]
@Katka92 @rhopp please review

File: cico/config_rh_che_automated [hunk 1]
```diff
@@ -0,0 +1,11 @@
+set +x
```
Comment (rhopp) [inline cico/config_rh_che_automated]: I guess this file slipped here by accident and should be part of another PR, right?

Comment (ScrewTSW) [reply cico/config_rh_che_automated]: yes, sorry about that.

**Comment by ScrewTSW** [comment]
[test]

**Comment by Katka92** [comment]
Use variable to know which cluster will be used. Rewrite skripts for generating logs for zabbix to use this variable.
